esrally/track/loader.py

```diff
@@ -1497,6 +1496,8 @@ def parse_parallel(self, ops_spec, ops, challenge_name):
                         f"'parallel' element for challenge '{challenge_name}' contains multiple tasks with the "
                         f"name '{completed_by}' marked with 'completed-by' but only task is allowed to match."
                     )
+                elif task.any_completes_parent:
+                    completion_task = "any"
```

Comment:
We use `completion_task` only as a flag whether there is a completion task so a specific name is actually not required. Can we refactor this to only set a flag instead in both cases (any task completes the parallel structure and specific task completes the parallel structure)?